fe/src/composables/useUtils.ts

```diff
@@ -7,5 +7,18 @@ export const useUtils = () => {
     }
   };
 
-  return { truncate };
+  const middleTruncate = (
+    address: string,
+    startLength: number,
+    endLength: number
+  ) => {
```

Comment:
Pls put return type as well
```suggestion
  ): string => {
```

Reply:
added return type `string` to the two functions in `useUtils.ts`